disk/tests/test_unit.py

```diff
@@ -155,3 +156,29 @@ def test_min_disk_size(aggregator, gauge_metrics, rate_metrics):
         aggregator.assert_metric_has_tag(name, 'device:{}'.format(DEFAULT_DEVICE_NAME))
 
     aggregator.assert_all_metrics_covered()
+
+
+@pytest.mark.skipif(sys.platform != 'linux', reason='disk labels are only available on Linux')
+@pytest.mark.usefixtures('psutil_mocks')
+def test_labels_from_blkid_cache_file(aggregator, instance_blkid_cache_file):
+    """
+    Verify that the disk labels are set with when the blkid_cache_file option is set
+    """
+    c = Disk('disk', {}, [instance_blkid_cache_file])
+    c.check(instance_blkid_cache_file)
+    for metric in EXPECTED_METRICS:
```

Comment:
It seems this is more appropriate (below), since `EXPECTED_METRICS` also contains metrics other  than `/dev/sda1`.

```
    for name in chain(gauge_metrics, rate_metrics):
        aggregator.assert_metric(name, tags=['device:/dev/sda1', 'label:MYLABEL'])
```

same comment for next test

Reply:
Example from: https://github.com/DataDog/integrations-core/blob/1bf1bcf5f4af4a54f16bf45ad76d55c4f364ce05/disk/tests/test_check.py#L16-L17